pkg/pool-weighted/test/IndexPool.test.ts

```diff
@@ -71,7 +79,7 @@ describe('IndexPool', function () {
   });
 
   describe('weights and scaling factors', () => {
-    for (const numTokens of range(2, MAX_TOKENS + 1)) {
+    for (const numTokens of range(3, MAX_TOKENS + 1)) {
```

Comment:
this dynamically creating test cases is really cool, I might use it to refactor the tests for IndexPoolUtils

Reply:
I didn't put that in there actually, Konstantin might have? I just adapted it for the 3 tokens minimum rule